src/data/markdown/docs/02 javascript api/07 k6-experimental/02 grpc/20 Client/20-Client-connect-connect-address-params.md

```diff
@@ -50,3 +62,71 @@ export default () => {
 };
 ```
 </div>
+
+<div class="code-group" data-props='{"labels": ["Different TLS settings"], "lineNumbers": [true]}'>
+
+```javascript
+import grpc from 'k6/experimental/grpc';
+import { check } from 'k6';
+
+// note: the services in this example don't exist. If you would like 
+// to run this example, make sure to replace the URLs, and
+// the cacerts, cert, key, and password variables.
+const params = {
+  'foo1.grpcbin.test.k6.io:9001': {
+    plaintext: false,
+    tls: {
+      cacerts: [open('cacerts0.pem')],
+      cert: open('cert0.pem'),
+      key: open('key0.pem'),
+    },
+  },
+  'foo2.grpcbin.test.k6.io:9002': {
+    plaintext: false,
+    tls: {
+      cacerts: open('cacerts1.pem'),
+      cert: open('cert1.pem'),
+      key: open('key1.pem'),
+      password: 'cert1-passphrase',
+    },
+  },
+};
```

Comment:
This was briefly discussed in the original PR - the certificates and keys are likely going to be big enough that loading each and everyone for each VU will be wasteful.

As such I would recommend using SharedArray to anyone who will use this to have different certificates per connection/VU. 

I would also prefer if any examples we provide follow this advice as users commonly copy paste examples as-is.